test_run.lua

```diff
@@ -40,8 +40,12 @@ local function get_param(self, node, param)
 end
 
 local function get_lsn(self, node, sid)
-    local nodes = self:get_param(node, 'vclock')
-    return tonumber(nodes[1][tonumber(sid)])
+    local node = self:get_param(node, 'vclock')[1]
+    if node == nil then
```

Comment:
Since it is not a vclock component, but a full vclock, the change does not fix the issue. I described it in my [comment](https://github.com/tarantool/test-run/pull/269#issuecomment-789373803).

Reply:
Thanks for the explanation, I've moved the changes from get_lsn to wait_lsn as you suggested.